src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -615,7 +632,16 @@ export class RoslynLanguageServer {
             childProcess = cp.spawn(serverPath, args, cpOptions);
         }
 
-        return childProcess;
+        //Log any stderr/stdout messages in our output channel.
+        childProcess.stderr.on('data', (data: { toString: (arg0: any) => any }) =>
+            _channel.append('[stderr]' + (isString(data) ? data : data.toString(this._encoding)))
```

Comment:
What would 'data' be that _isn't_ a string here?

Reply:
It's the encoded data buffer!